packages/foreman/foreman/foreman.spec

```diff
@@ -987,6 +994,9 @@ exit 0
 %systemd_postun_with_restart %{name}.service
 
 %changelog
+* Wed May 06 2020 Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl>
```

Comment:
The version and release are missing at the end here

Reply:
Odd - I used obal for it.

Reply:
And outdated version? It had bugs around fetching those on Py3 installs **whistle**

Reply:
I first had a macro error (`%end` instead of `%endif`) so it probably failed to parse the version out of it.